### ISSUE TITLE
Why is it also prompted that the component is not being used, but I have correctly bound the is attribute

### ISSUE THREAD

**Issue by Xavier0818** [vuejs/eslint-plugin-vue#657]
**Tell us about your environment**	
* **ESLint version:5.8.0
* **eslint-plugin-vue version:5.0.0-0	
* **Node version:10.9.0

**Please show your full configuration:**

```json

```

**What did you do?**	

```vue
module.exports = {
  root: true,
  env: {
    node: true
  },
  'extends': [
    'plugin:vue/recommended'
  ],
  rules: {
    'no-console': 'off',
    'no-debugger': 2,
    'vue/html-self-closing': 'warning'
  },
  parserOptions: {
    parser: 'babel-eslint'
  }
}
```

**What did you expect to happen?**

**What actually happened?**
![img](https://raw.githubusercontent.com/Xavier0818/damai/master/public/img/%E5%BE%AE%E4%BF%A1%E6%88%AA%E5%9B%BE_20181113111728.png)

**Comment by ota-meshi** [comment]
Hello!
I think this error will be resolved if you change as follows.

```vue
<template>
  <div class="layout">
    <component :is="'Header'">
      <component :is="'searchBar'" />
    </component>
    <component :is="'appMain'" />
  </div>
</template>
```

**Comment by armano2** [comment]
Header, searchBar and appMain are objects not strings

**Comment by Xavier0818** [comment]
> Hello!
> I think this error will be resolved if you change as follows.
> 
> ```
> <template>
>   <div class="layout">
>     <component :is="'Header'">
>       <component :is="'searchBar'" />
>     </component>
>     <component :is="'appMain'" />
>   </div>
> </template>
> ```

[https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/no-unused-components.md](url)

but The example is same as i writed

**Comment by Xavier0818** [comment]
> Header, searchBar and appMain are objects not strings
> 
> please read: https://vuejs.org/v2/api/#is

it says the attribute could be Objects or String

**Comment by armano2** [comment]
can you update to `5.0.0-beta.4` this issue got fixed in
https://github.com/vuejs/eslint-plugin-vue/commit/2049c5c0187a7d2c45b71d3a28611f81b1d6169b#diff-0af8dcbaa4e50eb0420c85c56dfedbf6

```
* Fix custom component detection
* Add ignoreWhenBindingPresent setting to no-unused-components
* Make ignoreWhenBindingPresent setting true by default in no-unused-components rule
```

**Comment by ota-meshi** [comment]
> but The example is same as i writed

is not same.

```vue
<component :is="Header">
<component :is="'Header'">
```

**Comment by Xavier0818** [comment]
> > but The example is same as i writed
> 
> is not same.
> 
> ```
> <component :is="Header">
> <component :is="'Header'">
> ```

yes! it works! thanks!
